gradle/versions.gradle

```diff
@@ -64,5 +68,9 @@ dependencyManagement {
     dependency 'org.web3j:core:4.5.14'
 
     dependency 'org.zeroturnaround:zt-exec:1.11'
+
+    dependency 'info.picocli:picocli:4.5.1'
```

Comment:
don't think signers should depend upon Picocli - this should be a utility library

Reply:
it is also used in tests.

Reply:
Moved it directly into module's build file as it is mainly required at test runtime to execute the yubishell simulator.